sources/Debate-frontend/src/app/account/account.component.html

```diff
@@ -0,0 +1,31 @@
+<div class="accountContent">
+    <mat-card class="accountCard" >
+    <div *ngIf="userDto.login === ''; then thenBlock else elseBlock"></div>
```

Comment:
thenBlock and elseBlock could have more meaningful names, i.e loginSection, accountSection